docs/governance/community-guidelines/content-standards-and-practices/threat-definitions.md

```diff
@@ -4,23 +4,44 @@ A threat definitions document is a detailed specification of potential security
 
 Each threat definition should be created for a service in the CCC Taxonomy, with each threat mapped to a specific feature and linked to relevant threat models.
 
+## Common vs Specific Threats
+
+In order to streamline maintenance, the CCC project maintains a list of [common threats].
+
+In each service category's `threats.yaml` document, common threats are referenced in a list of IDs in the top-level value `common-threats`.
+
+In the release pipeline, our [delivery tooling] will compile the common threats into the document alongside the specific threats. In the final output, the only difference in presentation of the threats will be the unique identifier.
+
 ## Threat Definition Format
 
-In order to create a cohesive standard that is readily useful to end users, threats must be indistinguishable from each other in format, style, and tone. A review from the [Communications WG] is recommended, but not required, in cases where additional support is needed to match the writing style and tone.
+In order to create a cohesive standard that is readily useful to end users, threats must be indistinguishable from each other in format, style, and tone. As such, all threats must match the layout presented in the [threats template] prior to release.
+
+A review from the [Communications WG] is recommended, but not required, in cases where additional support is needed to match the writing style and tone.
+
+[!NOTE] The list of common threats follows a similar but unique format, which can be found in the [common threats] file.
 
-As such, all threats must match the layout presented in the [threat template](../templates/threats.yaml) prior to release.
+### Common Threat References
```

Comment:
As discussed, it makes sense that there's a common threat if there's a common control. Only very slight concern is that most threat modelling output I've seen so far has threats defined by including [sub-techniques](https://attack.mitre.org/resources/faq/) or[ procedures](https://attack.mitre.org/resources/faq/) specific to the service/system being threat modelled. For common threats, we wouldn't be including service/system-specific detail on sub-techniques or procedures, which (I think) means we'd be aligning more with common [techniques](https://attack.mitre.org/techniques/enterprise/) when defining common threats (the sub-techniques/procedures would only come through in the behavioural tests for the controls) . 

With the common threats approach, it'd be interesting to take a look at whether we'd just end up rephrasing an existing MITRE technique when defining our threats - and if so are we really benefiting from that. Taking a threat that we recently have defined at the service level - e.g. https://github.com/finos/common-cloud-controls/blob/main/services/storage/object/threats.yaml#L4C9-L4C25 and then making that "common" might result in something like: _Attacker intercepts data in transit_. If we compare/map to MITRE techniques they would probably be:

- [Adversary-in-the-Middle](https://attack.mitre.org/techniques/T1557/) 
- [Data from Cloud Storage ](https://attack.mitre.org/techniques/T1530/)